mifosng-android/src/main/java/com/mifos/services/API.java

```diff
@@ -303,6 +308,26 @@ public void getCenterList(@Query("dateFormat") String dateFormat, @Query("locale
         public void getAllAccountsOfClient(@Path("clientId") int clientId, Callback<ClientAccounts> clientAccountsCallback);
 
     }
+    public interface SurveyService {
+
+        @Headers({ACCEPT_JSON, CONTENT_TYPE_JSON})
+        @GET(APIEndPoint.SURVEYS)
+        public void getAllSurveys(Callback<List<Survey>> callback);
+
+      //  @Headers({ACCEPT_JSON, CONTENT_TYPE_JSON})
+       // @POST(APIEndPoint.SURVEYS)
+       // void createSurvey(@Body SurveyPayload surveyPayload, Callback<Survey> callback);
```

Comment:
Please remove the Commented out code. You can put this in a separate branch locally.